### PR TITLE
SUP-10985 #comment support native HLS text tracks

### PR DESCRIPTION
**Pull request by yairans**

### CODE REVIEW

File: modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js [hunk 1]
```diff
@@ -1556,6 +1571,36 @@
...
 		},
+		onSwitchTextTrack: function (event, data) {
+			var vid = this.getPlayerElement();
+			var textTracks = vid.textTracks;
+			if (textTracks && textTracks.length) {
+				if (!data) {
+					var activeSubtitle = this.getActiveSubtitle(textTracks);
+					if (activeSubtitle) {
+						activeSubtitle.mode = 'hidden';
+						this.log('onSwitchTextTrack disable subtitles');
```
Comment (itaykinnrot) [inline modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js]: we have this.log here? or mw.log?

File: modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js [hunk 2]
```diff
@@ -1452,20 +1453,34 @@
...
+			                    'srclang': textTrack.label,
+			                    'label': textTrack.label,
+			                    'title': textTrack.label,
+			                    'id': textTrack.id,
+			                    'index': i
+		                    });
+	                    }
+	                    textTrack.mode = 'hidden';
+                    }
+                    if (textTracksData.languages.length) {
```
Comment (itaykinnrot) [inline modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js]: Check that the object is not null before checking length

Comment (yairans) [inline modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js]: not necessary. i just created it some lines above with languages as empty array (#1462).

File: modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js (hunk 1 above)
Comment (yairans) [inline modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js]: both of them. for simple string used this, for string and object used mw

**Pull request merged**